CONTRIBUTING.md

```diff
@@ -1,38 +1,37 @@
 
 ## Contributor License Agreement
 
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution;
-this simply gives us permission to use and redistribute your contributions as
-part of the project. Head over to <https://cla.developers.google.com/> to see
-your current agreements on file or to sign a new one.
+Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution; this simply gives us permission to use and redistribute your contributions as part of the project. Head over to [CLA](https://cla.developers.google.com/) to see your current agreements on file or to sign a new one.
 
-You generally only need to submit a CLA once, so if you've already submitted one
-(even if it was for a different project), you probably don't need to do it
-again.
+You generally only need to submit a CLA once, so if you've already submitted one (even if it was for a different project), you probably don't need to do it again.
 
 ## Code Reviews
 
-All submissions, including submissions by project members, require review. We
-use GitHub pull requests for this purpose. Consult
-[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
-information on using pull requests.
+All submissions, including submissions by project members, require review. We use GitHub pull requests for this purpose. Consult [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more information on using pull requests.
 
 ## Community Guidelines
 
-This project follows [Google's Open Source Community
-Guidelines](https://opensource.google/conduct/).
+This project follows [Google's Open Source Community Guidelines](https://opensource.google/conduct/).
 
 ## Commit Message Conventions
+
 - Start with a short summary (50 characters or less) of the changes made.
 - Use the present tense and imperative mood.
 - Separate the summary from the body of the message with a blank line.
 - Use the body to explain what and why changes were made, as well as any necessary details.
-- Addionally, you can consider using [semantic commit messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716?permalink_comment_id=3867882) like "feat:", "docs:", etc. which will provide additional context to the commit message.
+- Additionally, you can consider using [semantic commit messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716?permalink_comment_id=3867882) like "feat:", "docs:", etc. which will provide additional context to the commit message.
+
+| Commit Type | Description |
+| ---- | ---- |
+| `feat` | New feature or functionality added |
+| `fix` | Bug fix |
+| `docs` | Changes to documentation |
+| `test` | Adding or updating tests |
+| `chore` | Maintenance tasks such as refactoring, dependencies updates, or removing unused code |
```

Comment:
What does `chore` mean? Would it be better to just use `misc`?

Reply:
Using "chore" as a commit type can help keep your commit messages organized and make it easier for other contributors and reviewers to understand the purpose of your changes.

While it is theoretically feasible to use "misc" instead of "chore," I advise sticking with the standard commit types and saving "chore" for maintenance work. This will keep the commit messages consistent and make it simpler for other contributors to grasp the rationale behind your modifications.

Reply:
so do you think its better to change it ?

Reply:
Nah, makes sense.